docs/how-to/rest-apis.txt

```diff
@@ -0,0 +1,95 @@
+.. _rest-api-getting-started:
+
+Getting Started with REST APIs
```

Comment:
This section describes how our APIs are made, and I think we instead want to jump right in to how to use them.  Maybe put the Swagger UI section first?  I would really focus on consumption by people who probably don't know what all of these tools we used to make the endpoints are.